Sources/PersistenceService.swift

```diff
@@ -3,53 +3,55 @@ import Foundation
 struct PersistenceService: InterestPersistable {
 
     let service: UserDefaults
+    private let prefix = "com.pusher.sdk"
 
     func persist(interest: String) -> Bool {
         guard !self.interestExists(interest: interest) else { return false }
 
-        service.set(interest, forKey: interest)
+        service.set(interest, forKey: self.prefixInterest(interest))
         return true
     }
 
-    func persist(interests: Array<String>) -> Bool {
-        guard !self.interestsExists(interests: interests) else { return false }
-
-        service.set(interests, forKey: "subscriptions")
-        return true
+    func persist(interests: Array<String>) {
+        self.removeAll()
+        for interest in interests {
+            _ = self.persist(interest: interest)
```

Comment:
Do you still need the `_ =`?

Reply:
It would work but compiler would show the following warning `Result of call to 'persist(interest:)' is unused`.

By adding `_ =` I want to stress my intention that I'm aware that `persist`  method is returning a value (`Bool`) but I'm intentionally ignoring it.

Reply:
If you're so inclined you could add a `@discardableResult` to the `persist` function and then you wouldn't need to explicitly ignore the return value with the `_`. Not sure what you want the default to be though, so maybe that's hiding something that you'd want the default case not to miss.

Reply:
This is a special use case, I need result in all other cases.